configure.sh

```diff
@@ -145,7 +145,7 @@ Options:
 #
 # The option parsing function. Uses getopts, a bash built-in function.
 #
-while getopts "d:b:l:hv" opt
+while getopts "d:b:l:hvs" opt
 do
```

Comment:
Where is the ‘s’ case handled?

Reply:
Sorry for having taken sooo long to answer...
Actually, the `s` case is already handled in the `case` matching below (on [line 172](https://github.com/javalib-team/sawja/pull/7/files#diff-0ba17b1855b0f97042c31b2b2c2fd2b075fa355a840b152f1959daf2a755266eL172) to be precise), but the `getopts`  optstring hadn't been updated accordingly, which rendered the case unreachable.